app/javascript/packs/hero-rotation.js

```diff
@@ -32,25 +38,40 @@ window.addEventListener('load', () => {
 
   const renderNextImage = images => {
     const [activeElem, inactiveElem] = getImageElements();
-    const nextImage = images[getNextImage(images, activeElem.style.backgroundImage)];
+    const nextImage =
+      images[getNextImage(images, activeElem.style.backgroundImage)];
     inactiveElem.style.backgroundImage = nextImage;
 
     inactiveElem.classList.add('active');
     activeElem.classList.remove('active');
 
     if (checkWidth()) {
       setTimeout(() => {
-        renderNextImage(images)
+        renderNextImage(images);
       }, heroBackgroundSpeed);
     }
   };
 
-  $.get(`${getBaseURL()}/hero_images`).done(response => {
-    const images = response.map(res => `url("${res.included[0].attributes.hero_image_thumbnail_url}")`);
-    if (checkWidth()) {
-      setTimeout(() => {
-        renderNextImage(images);
-      }, heroBackgroundSpeed)
+  let request = new XMLHttpRequest();
+  request.open('GET', getBaseURL() + '/hero_images', true)
```

Comment:
While it uses the same name for the variable, my understanding is `XMLHttpRequest()` is part of the JavaScript core library, while `request` is a module for node. Axios itself is a wrapper for XMLHttpRequest() in some instances, for example: https://github.com/axios/axios/blob/dc4bc49673943e35280e5df831f5c3d0347a9393/lib/adapters/xhr.js

Reply:
Oops, this was silly of me---I missed the `let` and thus didn't realize you were instantiating a **variable** called `request` instead of calling on the library.